packages/web3-eth-accounts/CHANGELOG.md

```diff
@@ -149,4 +149,8 @@ Documentation:
 
 -   Fixed `recover` function, `v` will be normalized to value 0,1 (#6344) 
 
+### Changed
```

Comment:
if accounts version is not bumped there shouldn't be any change in its file

Reply:
bumped account ver